marlowe-playground-client/grammar.ne

```diff
@@ -119,7 +119,7 @@ bounds
 
 bound
    -> hole {% ([hole]) => hole %}
-    | "Bound" someWS number someWS number {% ([{line, col},,bottom,,top]) => opts.mkTerm(opts.mkBound(bottom)(top))({startLineNumber: line, startColumn: col, endLineNumber: top.line, endColumn: top.col + top.value.toString().length}) %}
+    | "Bound" someWS number someWS number {% ([{line, col},,bottom,,top]) => opts.mkTerm(opts.mkBound(bottom)(top))({startLineNumber: line, startColumn: col, endLineNumber: top.line, endColumn: top.col + top.toString(10).length}) %}
```

Comment:
what's this doing?

Reply:
`top` is a `BigInteger`. There's an assumption here about its internal structure (that it has a `value` property), which we have no guarantee of. The best way to get the length of the string version of the value is to call its `toString` function.

Why the `10`? That's `BigInteger` "format in base 10" call.

Reply:
but `value` is a property of a token, `top` is a token type which should always have a `value` property. I think this code will call `toString` on a javascript object that represents tokens

Reply:
Ah, you're right. 🤦 

Thanks. :-)